### PR TITLE
docs(tile-converter): Improve dockerhub links

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

**Review by Tamrat-B — APPROVED** [review]
Looks good.

File: modules/tile-converter/docs/cli-reference/tile-converter.md [hunk 1]
```diff
@@ -36,10 +48,10 @@ $ npx tile-converter --help
...
 | egm                  | \*                        | \*                        | location of the Earth Gravity Model (\*.pgm) file to convert heights from ellipsoidal to gravity-related format, default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib https://geographiclib.sourceforge.io/html/geoid.html |
-| token                | \*                        |                           | Token for Cesium ION tilesets authentication                                                                                                                                                                                                         |
+| token                | \*                        |                           | Token for I3S or Cesium ION tileset authentication.                                                                                                                                                                                                         |
```
Comment (belom88) [inline modules/tile-converter/docs/cli-reference/tile-converter.md:54]: `token` is not supported in `I3S`->`3DTiles` conversion

Comment (dryabinin-actionengine) [reply modules/tile-converter/docs/cli-reference/tile-converter.md]: Removed I3S from here.

**Comment by dryabinin-actionengine** [comment]
@ibgreen Added one more commit with token option description changes + badges fix.
[Updated md file](https://github.com/visgl/loaders.gl/blob/9778a3a189d308e33801274241ff1fbbae88cdef/modules/tile-converter/docs/cli-reference/tile-converter.md)

**Comment by ibgreen** [comment]
Still an issue with the dockerhub badge I think: https://github.com/visgl/loaders.gl/pull/1729/files?short_path=132663c#diff-132663c9eee09f08c2d9c543246d2281fb1918b36d1c82cec1f5c4783f2470ca

**Comment by dryabinin-actionengine** [comment]
> Still an issue with the dockerhub badge I think: https://github.com/visgl/loaders.gl/pull/1729/files?short_path=132663c#diff-132663c9eee09f08c2d9c543246d2281fb1918b36d1c82cec1f5c4783f2470ca

FIxed.

**Pull request merged**